worker/cluster_worker.ml

```diff
@@ -73,11 +73,12 @@ type t = {
   allow_push : string list;            (* Repositories users can push to *)
 }
 
-let docker_push ~switch ~log t hash { Cluster_api.Docker.Spec.target; user; password } =
+let docker_push ~switch ~log t hash { Cluster_api.Docker.Spec.target; auth } =
   let repo = Cluster_api.Docker.Image_id.repo target in
   let target = Cluster_api.Docker.Image_id.to_string target in
-  Log.info (fun f -> f "Push %S to %S as user %S" hash target user);
-  Log_data.info log "Pushing %S to %S as user %S" hash target user;
+  let pp_auth ppf = Option.iter (fun (user, _) -> Format.fprintf ppf " as user %S" user) in
```

Comment:
Minor, but the rest of the code is using `Fmt`, so maybe:
```suggestion
  let pp_auth = Fmt.option (Fmt.using fst (Fmt.fmt " as user %S"))) in
```
or
```suggestion
  let pp_auth = Fmt.option (fun f (user, _) -> Fmt.pf f " as user %S" user) in
```

Reply:
Nice - I'm (alas) not familiar enough with `Fmt`!